docs/content/querying/filters.md

```diff
@@ -151,6 +151,97 @@ The grammar for a IN filter is as follows:
 }
 ```
 
+### Bound filter
+
+Bound filter can be used to filter by a comparing dimension values to an upper value or/and a lower value. 
+By default Comparison is string based and **case sensitive**.
+To use numeric comparison you can set `alphaNumeric` to `true`.
+By default the bound filter is a not a strict inclusion `inputString <= upper && inputSting >= lower`.
+  
+Note that if `alphaNumeric` is true, the column values that are not convertibles to `int` numbers will be treated as out of range.
+
```

Comment:
no pitfalls, but, not sure if person just reading documentation would understand what "out of range" meant in this context, so asked for examples, would be a nice to have.